apps/web/src/lib/query.ts

```diff
@@ -1,32 +1,42 @@
+import { isNaN } from "lodash";
+
 const rateLessonQueryKeys = {
   lessonId: "rate-lesson-id",
+  lessonStart: "rate-lesson-start",
   tutorId: "rate-tutor-id",
   tutorName: "rate-tutor-name",
 } as const;
 
 export function asRateLessonQuery({
   lessonId,
+  lessonStart,
   tutorId,
   tutorName,
 }: {
   lessonId: number;
+  lessonStart: string;
```

Comment:
`start` will deliver the same meaing as `lessonStart`. 

```suggestion
  start: string;
```

> [!NOTE]
> The url key can state `rate-lesson-start`